python_modules/dagster/dagster/_config/pythonic_config/conversion_utils.py

```diff
@@ -101,12 +101,15 @@ def _convert_pydantic_field(
     if get_origin(pydantic_field.annotation) == Literal:
         return _convert_typing_literal_field(pydantic_field)
 
-    field_type = pydantic_field.annotation
+    field_type = (
+        pydantic_field.annotation
+    )  # here by just passing the field_type is where we lose any default values
     if safe_is_subclass(field_type, Config):
         inferred_field = infer_schema_from_config_class(
```

Comment:
My thought was that we'd pass the default values to `infer_schema_from_config_class` and then propogate those down the recursion until they reach the intended non-Config fields. We'd have to deal with overrides, but that seems simple enough - the default set at the higher level would override one set on a lower model.